Connectivity review (#1047)
ARMmbed/Handbook: docs/reference/technology/connectivity/securesocket.md

socket->send(data, size);
```

> Please note that internal TLS structures require over 1 KB of RAM, so you need to allocate each TLS Socket from the heap by using the `new` command, instead of using stack or statically allocating it.

Author: ```suggestion
<span class="notes">**Note:** Internal TLS structures require over 1 KB of RAM. Allocate each TLS Socket from the heap by using the `new` command, instead of using stack or statically allocating it.</span>
```

virtual nsapi_error_t connect(const SocketAddress &address);
```

> The code above initiates the TCP connection and continues to TLS handshake. If [transport mode](#transport-modes) is either `TRANSPORT_KEEP` or `TRANSPORT_CLOSE`, TCP is assumed to be open and state directly goes into TLS handshake. This is currently forced to blocking mode. After successfully connecting, you can set it to nonblocking mode:

Author: ```suggestion
The code above initiates the TCP connection and continues to the TLS handshake. If [transport mode](#transport-modes) is either `TRANSPORT_KEEP` or `TRANSPORT_CLOSE`, TCP is assumed to be open and state directly goes into TLS handshake. This is currently forced to blocking mode. After connecting, you can set it to nonblocking mode:
```
